tests/storage/migration.go

```diff
@@ -333,6 +333,46 @@ var _ = SIGDescribe("[Serial]Volumes update with migration", Serial, func() {
 			waitForMigrationToSucceed(vm.Name, ns)
 		})
 
+		It("should migrate the source volume from a block source and filesystem destination DVs", func() {
```

Comment:
So following the discussion in https://github.com/kubevirt/kubevirt/pull/12818#discussion_r1763468740 I don't see a way to achieve this for this test (have to do block->fs, can't really generalize with the dv.spec.storage API)

Reply:
ok thanks